resources/EPvalidator.rb

```diff
@@ -307,15 +307,15 @@ def self.run_validator(hpxml_doc)
       "/HPXML/Building/BuildingDetails/Systems/HVAC/HVACPlant/CoolingSystem[CoolingSystemType='central air conditioner']" => {
         "../../HVACDistribution[DistributionSystemType/AirDistribution | DistributionSystemType[Other='DSE']]" => one_or_more, # See [HVACDistribution]
         "DistributionSystem" => one,
-        "AnnualCoolingEfficiency[Units='SEER']/Value" => one,
         "CoolingCapacity" => one, # Use -1 for autosizing
+        "AnnualCoolingEfficiency[Units='SEER']/Value" => one,
```

Comment:
Since order matters in HPXML elements, I like to try to preserve the order in this file. It doesn't have any impact on functionality, but helps any software developers referencing this file when producing HPXML files.